apps/studio/pages/new/v2/[slug].tsx

```diff
@@ -0,0 +1,1211 @@
+  instanceSize: z.string(),
+  dataApi: z.boolean(),
+  useApiSchema: z.boolean(),
+  postgresVersionSelection: z.string(),
+  useOrioleDb: z.boolean(),
+  sql: z.string().optional(),
+})
+
+export type CreateProjectForm = z.infer<typeof FormSchema>
+
+export const TABLE_NODE_WIDTH = 640
+export const TABLE_NODE_ROW_HEIGHT = 80
+
+interface SupabaseService {
+  name: 'Auth' | 'Storage' | 'Database' | 'Edge Function' | 'Cron' | 'Queues' | 'Vector'
+  reason: string
+}
+
+const WizardForm = () => {
+  const router = useRouter()
+  const { slug, projectName } = useParams()
+
+  const [showAdvanced, setShowAdvanced] = useState<Boolean>(false)
+  const [services, setServices] = useState<SupabaseService[]>([])
+  const [sqlStatements, setSqlStatements] = useState<string[]>([])
+  const [showVisual, setShowVisual] = useState(false)
+
+  const [step, setStep] = useState(1)
+  const [aiDescription, setAiDescription] = useState('')
+  const [formTitle, setFormTitle] = useState('Create a new project')
+  const [formDescription, setFormDescription] = useState(
+    'Get started by choosing how you want to create your project'
+  )
+
+  const projectCreationDisabled = useFlag('disableProjectCreationAndUpdate')
+  const projectVersionSelectionDisabled = useFlag('disableProjectVersionSelection')
+  const cloudProviderEnabled = useFlag('enableFlyCloudProvider')
+  const allowOrioleDB = useFlag('allowOrioleDb')
+  const { data: membersExceededLimit, isLoading: isLoadingFreeProjectLimitCheck } =
+    useFreeProjectLimitCheckQuery({ slug })
+
+  const [passwordStrengthMessage, setPasswordStrengthMessage] = useState('')
+  const [passwordStrengthWarning, setPasswordStrengthWarning] = useState('')
+
+  const { data: organizations, isSuccess: isOrganizationsSuccess } = useOrganizationsQuery()
+  const currentOrg = organizations?.find((o: any) => o.slug === slug)
+
+  const { data: orgSubscription } = useOrgSubscriptionQuery({ orgSlug: slug })
+
+  const { data: allOverdueInvoices } = useOverdueInvoicesQuery({
+    enabled:
+      orgSubscription !== undefined &&
+      !['team', 'enterprise'].includes(orgSubscription?.plan.id ?? ''),
+  })
+  const overdueInvoices = (allOverdueInvoices ?? []).filter(
+    (x) => x.organization_id === currentOrg?.id
+  )
+  const hasOutstandingInvoices = overdueInvoices.length > 0
+
+  const { data: allProjects } = useProjectsQuery({})
+  const organizationProjects =
+    allProjects?.filter(
+      (project) =>
+        project.organization_id === currentOrg?.id && project.status !== PROJECT_STATUS.INACTIVE
+    ) ?? []
+  const { data: defaultRegion, error: defaultRegionError } = useDefaultRegionQuery(
+    {
+      cloudProvider: PROVIDERS[DEFAULT_PROVIDER].id,
+    },
+    {
+      refetchOnMount: false,
+      refetchOnWindowFocus: false,
+      refetchInterval: false,
+    }
+  )
+
+  const {
+    mutate: createProject,
+    isLoading: isCreatingNewProject,
+    isSuccess: isSuccessNewProject,
```

Comment:
unused variables, although to double check if we're supposed to be using these for any UI responses/transitions?